backend/FwLite/FwLiteProjectSync.Tests/SyncTests.cs

```diff
@@ -137,6 +137,77 @@ await crdtApi.CreateEntry(new Entry()
                 .For(e => e.ComplexForms).Exclude(c => c.Id));
     }
 
+    [Fact]
+    public async Task PartsOfSpeechSyncBothWays()
+    {
+        var crdtApi = _fixture.CrdtApi;
+        var fwdataApi = _fixture.FwDataApi;
+        await _syncService.Sync(crdtApi, fwdataApi);
+
+        var noun = new PartOfSpeech()
+        {
+            Id = new Guid("a8e41fd3-e343-4c7c-aa05-01ea3dd5cfb5"),
+            Name = { { "en", "noun" } },
+            Predefined = true,
+        };
+        await fwdataApi.CreatePartOfSpeech(noun);
+
+        var verb = new PartOfSpeech()
+        {
+            Id = new Guid("86ff66f6-0774-407a-a0dc-3eeaf873daf7"),
+            Name = { { "en", "verb" } },
+            Predefined = true,
+        };
+        await crdtApi.CreatePartOfSpeech(verb);
+
+        await _syncService.Sync(crdtApi, fwdataApi);
+
+        var crdtPartsOfSpeech = await crdtApi.GetPartsOfSpeech().ToArrayAsync();
+        var fwdataPartsOfSpeech = await fwdataApi.GetPartsOfSpeech().ToArrayAsync();
+        crdtPartsOfSpeech.Should().BeEquivalentTo(fwdataPartsOfSpeech);
```

Comment:
let's be explicit that verb and noun are now in the list

Reply:
Done in commit 6d6f85af.